### PR TITLE
View users  #7 fixed #57 

### LINKED ISSUE #57: View users tables

### PR DESCRIPTION
**Pull request by amanshawar**
related to #7, fixed #57

- [ ] find all users in database BE
- [ ] show the users in table 
- [ ] update id from __id to id
 
-

### CODE REVIEW

File: client/src/components/pages/ViewUsers/index.js [hunk 1]
```diff
@@ -1,21 +1,85 @@
-import React, { Component } from "react";
-import {Link} from 'react-router-dom'
+import axios from 'axios';
+import React, {	Component} from "react";
+import {	Link} from 'react-router-dom'
```
Comment (mhmdtshref) [inline client/src/components/pages/ViewUsers/index.js]: Remove this space here, run `eslint:fix`.

File: client/src/components/pages/ViewUsers/index.js [hunk 2]
```diff
@@ -1,21 +1,85 @@
...
 import Header from "../../Header";
-import {StyledPage, StyledTilte} from "./index.style";
+import Table from "../../Table";
+import {	StyledPage,	StyledTilte} from "./index.style";
```
Comment (mhmdtshref) [inline client/src/components/pages/ViewUsers/index.js]: also this space here

File: src/controllers/user.controller.js [hunk 3]
```diff
@@ -14,4 +14,28 @@ const add = (req, res) => {
...
     });
 };
-module.exports = { add };
+
+
+const findAll = (req, res) => {
+  User.find({})
+    .exec((err, users) => {
+      if (err) {
+        res.json({ success: false, error: err.message });
```
Comment (mhmdtshref) [inline src/controllers/user.controller.js]: Out style on error response is like:
`{ success: false, err: err.message }`

File: src/controllers/user.controller.js [hunk 4]
```diff
@@ -14,4 +14,28 @@ const add = (req, res) => {
...
+      } else {
+        const data = users.map(user => ({
+          id: user._id, // eslint-disable-line no-underscore-dangle
```
Comment (mhmdtshref) [inline src/controllers/user.controller.js]: Did you tried to type `user.id` ?

File: client/src/components/pages/ViewUsers/index.js (hunk 1 above)
Comment (amanshawar) [reply client/src/components/pages/ViewUsers/index.js]: done

**Pull request merged**